scripts/start_server.sh

```
@@ -0,0 +1,29 @@
+cd /home/ubuntu/app
+
+# 1. S3에서 환경변수 파일 로드
+aws s3 cp s3://articker-codedeploy-artifacts-bucket/.env-prod .
+
+# EC2 메타데이터에서 현재 인스턴스의 고유 ID를 가져옵니다.
+INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
+# docker-compose가 사용할 수 있도록 환경 변수로 export 합니다.
+export INSTANCE_ID
+
+# 2. 배포된 파일에서 실행할 Docker 이미지 이름을 읽어옴
+export SPRING_APP_IMAGE=$(cat ./image_name.txt)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)